src/ClientData/include/ClientData/ScopeTree.h

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef ORBIT_GL_SCOPE_TREE_H_
-#define ORBIT_GL_SCOPE_TREE_H_
+#ifndef CLIENT_DATA_SCOPE_TREE_H_
+#define CLIENT_DATA_SCOPE_TREE_H_
 
 #include <cstdint>
 #include <memory>
```

Comment:
also add a orbit_client_data namespace, here and for block chain

Reply:
Should I create a namespace orbit_containers?

Reply:
yes please.